### DISCUSSION TITLE
TypeScript implementation "paseto-ts" on implementation list

### DISCUSSION THREAD

**Discussion by dawadam** [panva/paseto#57] [Ideas]
Hi, please add this TypeScript implementation on documentation : https://github.com/auth70/paseto-ts

(On implementation list in https://paseto.io/)

**Comment by panva** [comment]
Hi @dawadam 

I am not affiliated with the paseto.io website and have no control over maintaining its list of implementations.

**Comment by dawadam** [comment]
I posted the request in the wrong repository. :)
